src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -1,19 +1,67 @@
 package frc.robot.constants;
 
 import com.revrobotics.CANSparkBase.IdleMode;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
 import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
 import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.units.Angle;
 import edu.wpi.first.units.Distance;
 import edu.wpi.first.units.Measure;
 import edu.wpi.first.units.Units;
 import edu.wpi.first.units.Velocity;
 
-// IDs and stuff
+/**
+ * Software/hardware constants (e.g. CAN IDs, gear ratios, field measurements, etc.). For software
+ * configs @see RobotConfig
+ */
 public final class RobotConstants {
+
+  public final class Bindings {
+    public static final int kAimAmp = 4;
+    public static final int kAimSpeaker = 3;
+    public static final int kShoot = 1;
+    public static final int kShootReverse = 7;
+    public static final int kAimTrap = 2;
+    public static final int kStowShooter = 14;
+    public static final int kToggleFlywheel = 5;
+    public static final int kRetractShield = 10;
+    public static final int kExtendShield = 9;
+    public static final int kManualAdjustDown = 18;
+    public static final int kManualAdjustUp = 19;
+  }
+
+  public static final class VisionConstants {
+    public static final double camChassisXOffset = -1;
+    public static final double camChassisYOffset = -1;
+    public static final double camChassisZOffset = -1;
+
```

Comment:
why do we have vision files if the constants for them are not even set? if were going to implement during comp we should have these set or written down somewhere